CHANGELOG.md

```diff
@@ -2,7 +2,7 @@ CHANGELOG
 =========
 
 ## HEAD (Unreleased)
-_(none)_
+* Remove unused code [(#1021)](https://github.com/pulumi/pulumi-aws/pull/1021)
```

Comment:
I guess no one would have been able to use this module because of its name and hence we don't have to worry about listing this as a "breaking" change?

Reply:
Relatedly - we only need to add CHANGELOG entries for things that have a user-facing impact.  In this case, I actually think we can safely leave out of the CHANGELOG entirely.

Reply:
@praneetloke correct! @lukehoban yeah I was on the fence for this one, but that makes sense. removed it.